app/models/dialog_field_drop_down_list.rb

```diff
@@ -30,6 +38,18 @@ def refresh_json_value(checked_value)
     {:refreshed_values => refreshed_values, :checked_value => @value, :read_only => read_only?, :visible => visible?}
   end
 
+  def automate_output_value
+    return super unless multi_value?
+    a = @value.chomp.split(',')
```

Comment:
has to be: 
a = @value.blank? ? [] : @value.chomp.split(',')

otherwise it will break when there is a nil, which there is, when more selectboxes are linked